lib/MooX/Should.pm

```diff
@@ -2,9 +2,20 @@ package MooX::Should;
 
 # ABSTRACT: optional type restrictions for Moo attributes
 
+use version 0.77 ();
+
 use Moo       ();
 use Moo::Role ();
 
+our $USE_MOO_UTILS;
+
+BEGIN {
+    if( version->parse( Moo->VERSION ) >= version->parse('2.003006') ) {
+        $USE_MOO_UTILS = 1;
+        require Moo::_Utils;
+    }
+}
+
 use Devel::StrictMode;
 
 our $VERSION = 'v0.1.3';
```

Comment:
If you're going to use `version`, then you might as well use
```perl
our $VERSION = version->declare('v0.1.3');
```

Reply:
Done!